pegasus/forms/pd_workshop_survey.rb

```diff
@@ -76,6 +76,14 @@ def self.normalize(data)
     result
   end
 
+  def self.process_(form)
+    # Save this form id in the relevant dashboard pd_enrollment row
+    id = form[:id]
+    data = JSON.load(row[:data])
+    enrollment_id = data['enrollment_id_i']
+    DASHBOARD_DB[:pd_enrollments][enrollment_id].update(survey_id: id)
```

Comment:
This is the same as `DASHBOARD_DB[:pd_enrollments].find_by_id(enrollment_id).update(survey_id: id)`?